tkg/client/validate.go

```diff
@@ -1411,6 +1411,14 @@ func (c *TkgClient) ConfigureAndValidateCNIType(cniType string) error {
 	if _, ok := CNITypes[cniType]; !ok {
 		return errors.Errorf("provided CNI type '%s' is not in the available options: antrea, calico, none", cniType)
 	}
+
+	// _ALLOW_CALICO_ON_MANAGEMENT_CLUSTER must be true when cniType is calico
+	if cniType == "calico" {
+		allowCalicoType := os.Getenv("_ALLOW_CALICO_ON_MANAGEMENT_CLUSTER")
+		if allowCalicoType != "true" {
```

Comment:
Please double check the func `ConfigureAndValidateCNIType`. I think this will block workload cluster and pacific cluster creation as well

Reply:
As you can see all the calico related e2e tests in pr pipelines are all failing